app/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"database/sql"
+	"github.com/bxcodec/library/author/repository/postgres"
+	"github.com/bxcodec/library/book/delivery/http"
+	_book_repository "github.com/bxcodec/library/book/repository/postgres"
+	"github.com/bxcodec/library/book/usecase"
+	"github.com/bxcodec/library/mail"
+	"github.com/bxcodec/library/message_broker/rabbit"
+	"github.com/labstack/echo"
+	_ "github.com/lib/pq" // <------------ here
+	"log"
+	"time"
+)
+
+func main() {
+	dbConn, err := sql.Open(`postgres`,
+		"postgres://postgres:password@host.docker.internal:5432/postgres?sslmode=disable")
+
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = dbConn.Ping()
+	if err != nil {
```

Comment:
По кодстайлу лучше когда возвращается одна ошибка писать так:
```suggestion
	if err = dbConn.Ping(); err != nil {
```